### ISSUE TITLE
automate download from TEA

### ISSUE THREAD

**Issue by mscarey** [txappleseed/txappleseedmap#72]
Annual updates to the data used for the map involve downloading all 20 of the year's region files from http://rptsvr1.tea.texas.gov/adhocrpt/Disciplinary_Data_Products/Download_Region_Districts.html and pasting them together. This is tedious and it's a part of the process that's likely to generate errors. jgrenadier has looked at automating this with Selenium.

**Comment by jgrenadier** [comment]
I have uploaded my Selenium IDE scripts along with readme files that describe how to setup and run these scripts using the Selenium IDE.  Since there are multiple files, I put them in a directory called "selene" with in the scripts directory in github.  I'm a bit new to working with Github. I uploaded the files and created pull requests.  I think the ball is in your court to approve the changes. Let me know if there is more that I need to do to complete the commit to github.
